content/guides/developer-guides/delfin/_index.md

```diff
@@ -38,17 +38,167 @@ Existing Delfin Drivers for reference:
+
+    def list_volumes(self, ctx):
+      volume_list = []
+      for i in range(1, 5):
+          v = {
+              "name": "sample_vol_" + str(i),
+              "storage_id": self.storage_id,
+              "description": "Sample Volume",
+              "status": "normal",
+              "native_volume_id": "sample_original_id_" + str(i),
+              "wwn": "fake_wwn_" + str(i),
+              "total_capacity": 12,
+              "used_capacity": 2,
+              "free_capacity": 10,
+          }
+          volume_list.append(v)
+          return volume_list
+
+    def add_trap_config(self, context, trap_config):
+        pass
+
+    def remove_trap_config(self, context, trap_config):
+        pass
+
+    def parse_alert(self, context, alert):
+        pass
+
+    def clear_alert(self, context, alert):
+        pass
+
+    def list_alerts(self, context, query_para=None):
+        pass
+
+  ```
+
 * Ensure create storages API call from Delfin, can load the driver successfully.
 
+  ```bash
+  curl -H "Content-Type: application/json" -X POST http://localhost:8190/v1/storages -d '
+  {
+    "rest": {
+      "host": "127.0.0.1",
+      "port": "22",
+      "username": "username",
+      "password": "password",
+      "vendor": "sample_vendor",
+      "model": "sample_model"
+    }
+  }'
+
+  # Example Response:
+
+  '{"created_at": "2020-09-15T14:10:10.692091", "updated_at": null, "id": "849e0e90-35bd-4d23-8155-53a5a583bcbf", "name": "sample", "vendor": "sample_vendor", "description": "sample", "model": "sample_model", "status": "normal", "serial_number": 123, "firmware_version": "1.0.0", "location": "HK", "total_capacity": 123, "used_capacity": 0, "free_capacity": 123, "raw_capacity": 123, "subscribed_capacity": 123, "sync_status": "SYNCED", "deleted_at": null, "deleted": false}'
+
+
+  curl -H "Content-Type: application/json" -X GET http://localhost:8190/v1/storages
+
+  # Example Response:
+
+  {"storages": [{"created_at": "2020-09-15T14:11:05.611150", "updated_at": "2020-09-15T14:11:05.919163", "id": "2021529b-0235-4a75-b358-0779189d6463", "name": "fake_driver", "vendor": "fake_vendor", "description": "fake driver.", "model": "fake_model", "status": "normal", "serial_number": "1f8d0680-491d-4b13-9b08-996e1ef8ae6d", "firmware_version": "1.0.0", "location": "HK", "total_capacity": 1355, "used_capacity": 934, "free_capacity": 421, "raw_capacity": 2423, "subscribed_capacity": 3884, "sync_status": "SYNCED", "deleted_at": null, "deleted": false}, {"created_at": "2020-09-15T14:10:10.692091", "updated_at": "2020-09-15T14:10:10.970153", "id": "849e0e90-35bd-4d23-8155-53a5a583bcbf", "name": "sample", "vendor": "sample_vendor", "description": "sample", "model": "sample_model", "status": "normal", "serial_number": "123", "firmware_version": "1.0.0", "location": "HK", "total_capacity": 123, "used_capacity": 0, "free_capacity": 123, "raw_capacity": 123, "subscribed_capacity": 123, "sync_status": "SYNCED", "deleted_at": null, "deleted": false}]}
```

Comment:
Please format this to improve readability.

Reply:
Example for response for reference purpose, see above comment.